helpers/middlewares/limit_context.js

```diff
@@ -0,0 +1,26 @@
+let MAX_CONTEXT_COUNTER;
+let contextCounter = 0;
```

Comment:
can we use `broser.browserContexts.length` instead of a separate counter?

Reply:
No, we cannot. Consider the situation where we send many GoTo requests to the service without any delay. The service would not be able to instantly render all contexts. Since the function is asynchronous `browser.browserContexts.length` will still be `1` unless contexts are created. In this case we would have `1 > MAX_CONTEXT_COUNTER` which is false.